CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+# 0.27.0 (Aug 23, 2024)
+
+NOTES:
+
+**This release contains breaking and behavioural changes**
```

Comment:
obviously both spellings are right in their own way, but not sure which is right in this context. I guess more of a general docs style guide question that provider docs would follow?
```suggestion
**This release contains breaking and behavioral changes**
```

Reply:
Ha, my Canadian/British english (and spellchecker) is shining through. We use US spellings in our docs so I'll go with that: thanks for the catch!